### PR TITLE
Re-add Chinese translations

### PR DESCRIPTION
**Pull request by KnorpelSenf**

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
@roj1512 what can we do about the memory consumption? This did not happen back in the days, did it?

**Comment by roj1512** [comment]
@KnorpelSenf It’s my first time seeing it. Maybe we can do something to stop pending builds when there’s a new commit pushed. I think they do something like that in Deno’s https://github.com/denoland/deno repository.

**Comment by roj1512** [comment]
https://github.com/denoland/deno/blob/211f49619afd915a2ce56cf0b5da50f640c90f73/.github/workflows/ci.yml#L18

**Comment by roj1512** [comment]
Huh it seems not related

**Comment by KnorpelSenf** [comment]
Let's just throw more memory at the problem.

**Comment by CikiMomogi** [comment]
I have listed the pull requests that have the label "ready for translation" from commit 48aa7020 (the first commit since ZH was removed) to date. Hope this helps.

To do list:
- [ ] #530
- [ ] #502 
- [ ] #513 
- [ ] #506 
- [ ] #518 
- [ ] #536 
- [ ] #545 
- [ ] #442 
- [ ] #505 
- [ ] #511
- [ ] #558 
- [ ] #560 
- [ ] #561 
- [ ] #564 
- [ ] #563
- [ ] #557 
- [ ] #520 
- [ ] #497 
- [ ] #568 
- [ ] #572 
- [ ] #576 
- [ ] #556 
- [ ] #579 
- [ ] #578 
- [x] #594 
- [ ] #589 
- [x] #597 
- [x] #626 
- [x] #559 
- [x] #587 
- [x] #585 
- [ ] #642 
- [ ] #641 
- [ ] #590
- [ ] #625

File: site/docs/zh/plugins/chat-members.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+在许多情况下，bot 有必要了解给定聊天的所有用户的信息。
+不过，目前 Telegram Bot API 没有公开允许我们检索此信息的方法。
+
+这个插件来解决：自动监听 `chat_member` 事件并存储所有 `ChatMember` 对象。
+
+## 使用方法
+
+### 存储聊天成员
+
+你可以使用有效的 grammY [存储适配器](./session.md#known-storage-adapters) 或任何实现 [`StorageAdapter`](https://deno.land/x/grammy/mod.ts?s=StorageAdapter) 接口的类的实例。
```
Comment (CikiMomogi) [inline site/docs/zh/plugins/chat-members.md:17]: Broken link `./session.md#known-storage-adapters`

**Review by CikiMomogi — CHANGES_REQUESTED** [review]
#585

**Review by roj1512 — APPROVED** [review]
LGTM.

**Pull request merged**